config/initializers/yabeda.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+return unless Settings.dig(:scc, :metrics, :enabled)
+
+# Configure prometheus client
+Prometheus::Client.config.data_store = Prometheus::Client::DataStores::DirectFileStore.new(
+  dir: './tmp/prometheus/'
+)
+
+# Configure yabeda
+Yabeda.configure do
+  assign_labels = lambda {
+    default_tag :environment, Rails.env
+    default_tag :application, 'rmt'
+  }
+
+  group :rails, &assign_labels
+  group :rake, &assign_labels
+
+  group :rails do
+    counter :started_requests_total,
+            comment: 'A counter of the total number of HTTP requests rails has started to process.',
+            tags: %i[controller action format method]
+  end
+
+  group :rake do
+    gauge :task_started_at,
+          comment: 'Time when the task started: unix time with decimals',
+          tags: %i[task_name]
+
+    gauge :task_exit_status,
+          comment: 'Task exit status. 1 means OK, 0 means failed, not present means no data',
+          tags: %i[task_name]
+
+    gauge :task_duration_ms,
+          comment: 'Time taken running the task',
+          tags: %i[task_name]
+
+    gauge :task_finished_at,
+          comment: 'Time when the task finished: unix time with decimals',
+          tags: %i[task_name]
+  end
+end
+
+# Instrument the request from the start
+ActiveSupport::Notifications.subscribe 'start_processing.action_controller' do |*args|
+  # Match the same event as Yabeda
+  event = Yabeda::Rails::Event.new(*args)
+
+  Yabeda.rails.started_requests_total.tap do |metric|
+    labels = event.labels.slice(*metric.tags)
+
+    metric.increment(labels, by: 1)
+  end
+end
```

Comment:
Please remove this metric, it's a failed attempt to detect unusual behavior. At the end it ended up being closer to an `in-flight` request handler but not really :sweat_smile:

Reply:
I will leave this in for smoke test purposes.